src/script/main/app.ts

```diff
@@ -190,7 +190,7 @@ class App {
     private readonly core: Core,
     private readonly apiClient: APIClient,
   ) {
-    this.apiClient.on(APIClient.TOPIC.ON_LOGOUT, () => this.logout(SIGN_OUT_REASON.NOT_SIGNED_IN, false));
+    this.apiClient.on(APIClient.TOPIC.ON_LOGOUT, () => this.logout(SIGN_OUT_REASON.SESSION_EXPIRED, false));
```

Comment:
If the apiClient detects a 'logout' is means the token is expired (not that the user has logged out manually)